src/conda/env_ocn_surf_flux_diag.yml

```diff
@@ -0,0 +1,23 @@
+name: _MDTF_ocn_surf_flux_diag
```

Comment:
Remove this file from the PR. I put in PR to add metpy to the python3 base yaml, so you can use that in the POD settings file instead. It will save users the step of having to build a separate environment.